src/OpenLoco/Windows/ToolbarTop.cpp

```diff
@@ -154,7 +147,7 @@ namespace OpenLoco::Ui::Windows::ToolbarTop::Game
             return;
         }
 
-        if (!saveGameOpen())
+        if (!OpenLoco::Game::saveSaveGameOpen())
```

Comment:
Any particular reason for explicitly resolving `OpenLoco::`?

Reply:
Just applying the suggestion clang gave in https://github.com/OpenLoco/OpenLoco/runs/3365202052?check_suite_focus=true

Reply:
Oh, right, `Game` is also a namespace in `ToolbarTop`. Carry on then.